starter-code/index.js

```diff
@@ -1,11 +1,55 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
-};
+  constructor(length, item) {
+    this.length = 0;
+    this.item = [];
+  }
+
+  add(item) {
+    this.item.push(item);
+    this.length += 1;
+    this.item.sort(function(a, b) {
+      return a - b;
+    });
+  }
+
+  get(pos) {
+    var position = this.item[pos - 1];
+    return position;
+  }
+
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.item[this.length - 1];
+    }
+  }
+
+  min() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return this.item[0];
+    }
+  }
+
+  sum() {
+    var sum = 0;
+    this.item.forEach(function(oneItem) {
```

Comment:
normally, when calculating a sum, using a reduce is better because it signals better intent. It makes it easier for people to know exactly what you are doing when they first look at your code